### DISCUSSION TITLE
Add ccTLD and ISO 3166-1 alpha-3 shorthand for all notes where this applies?

### DISCUSSION THREAD

**Discussion by Inconsist** [axelboc/anki-ultimate-geography#641] [Ideas]
I already made a deck with all the ccTLD's I just dont know how to export the information to contribute it here.
The ISO codes I havent done jet, but both would be a nice to have for optional card types. 
how interested are people in this optional information?

**Comment by aplaice** [comment]
Thanks for the offer to contribute!

I don't think we'd want this in the main deck, but a variant deck with this info might be worth it! (It'd be valuable to know how many people might be interested!)

(In terms of process it's probably easiest if you either export as a CrowdAnki JSON or export notes as plain text.)

**Reply by Inconsist** [reply]
[deck.json](https://github.com/user-attachments/files/16479189/deck.json)

hope you can work with this, if you need anything else let me know. 

![image](https://github.com/user-attachments/assets/6406f80a-4b00-42c0-b624-100a1ef94890)

the last three entries are 
1.ccTLD
2.ISO 3166-1 alpha-2
3.2.ISO 3166-1 alpha-3

Im afraid there are some notes like northern cyprus, where there is no official information like this.

also I added the coat of arms/emblem for almost all my notes where there is one, if that is smoething people are intersted in.

![image](https://github.com/user-attachments/assets/15b8ad56-6b47-4e93-a9fd-7eb7433fa9c9)

**Reply by Inconsist** [reply]
I cant upload my media in these comments, but if its interesting to people I can commit the mdeia files on a later date.

**Reply by Inconsist** [reply]
[deck.json](https://github.com/user-attachments/files/16533990/deck.json)

I updated some things, I added outline shapes, and the telphone prefix for the countrys if those were available.
Once again, if you need the media, I can't upload it here, just tell me where/how to provide it.

![Screenshot 2024-08-07 223529](https://github.com/user-attachments/assets/0b8d910f-e237-4015-8aac-ed3c161d7f16)
![image](https://github.com/user-attachments/assets/31c7cc4b-9e82-4c38-9f8c-7708a14a956c)

**Reply by jbtodo** [reply]
@Inconsist  I came here looking for exactly what you did with country shapes! I have downloaded the json but the instructions on the home page warn against custom fields. I have used Anki for a while for language learning but do not know a lot of the technical aspects or anything about CrowdAnki yet. 
Any advice so I can start using those shapes?
